src/components/VideoPlayerContexts/VideoPopoverMenuContext.js

```diff
@@ -26,14 +27,14 @@ function VideoPopoverMenuContextProvider({children}) {
     );
 
     const downloadAttachment = useCallback(() => {
-        const sourceURI = currentlyPlayingURL.startsWith('blob:') || currentlyPlayingURL.startsWith('file:') ? currentlyPlayingURL : addEncryptedAuthTokenToURL(currentlyPlayingURL);
+        const sourceURI = isLocalFile ? currentlyPlayingURL : addEncryptedAuthTokenToURL(currentlyPlayingURL);
```

Comment:
Do we even still need to check `isLocalFile`, since you wouldn't be able to call this function when `isLocalFile` is `true`?

Reply:
I agree with you. it makes no sense